internal/test/morerequire/testdata/fake_envoy.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
I chose to use go:embed in testdata for a couple reasons.

1. this keeps things out of coverage results
2. it is easier than reading the file
3. `make format` will format this vs a string literal
4. it is too long for a string literal anyway